### DISCUSSION TITLE
is touchscreen mod meant to be enabled on multi

### DISCUSSION THREAD

**Discussion by junejsjsjdjjdjdjdjdj** [ppy/osu#26574] [Ideas] [answered]
on multiplayer pc gamers get pp
to get pp on moble you need touchscreen mod but in the multiplayer mode it does not enable 
bug or feature?

**Comment by bdach** [comment]
it is supposed to enable. looking into it

**Reply by bdach** [reply, marked as answer]
https://github.com/ppy/osu/pull/26575
